example/lib/main.dart

```diff
@@ -65,8 +62,9 @@ class _MyAppState extends State<MyApp> {
   Future<void> _signOut() async {
     String authToken;
     try {
-      authToken = await this.fma.signOut;
-    } on PlatformException catch(e) {
+      await this.fma.signOut;
+      authToken = "";
```

Comment:
single qoute string

Reply:
Didn't cleanup example project, has no analysis too; just fixed to accommodate lib changes and formatted.

Updated to single quote.